sdk/core/core-asynciterator-polyfill/package.json

```diff
@@ -19,9 +19,7 @@
   "main": "./dist-esm/index.js",
   "files": [
     "dist-esm/**/*.js",
-    "LICENSE",
```

Comment:
Was the license intentionally removed?

Reply:
It felt like unnecessary duplication, but I think I'm reconsidering to include it rather than exclude it from packages. I'll update this PR.

Reply:
Sweet! I think some license auditing tools might look for this file so it could be useful from that standpoint.

Reply:
should all be consistently including it now